repositories/loaders.json

```diff
@@ -26,5 +26,6 @@
   "webpack-contrib/polymer-webpack-loader",
   "babel/babel-loader",
   "postcss/postcss-loader",
```

Comment:
Somebody can create an issue, `postcss/postcss-loader` is now here https://github.com/webpack-contrib/postcss-loader

Reply:
will do

Reply:
Fixed here - https://github.com/webpack/webpack.js.org/pull/3909